backend/src/main/java/se/chalmers/dat265/group1/storage/PostgresSQLConnector.java

```diff
@@ -20,6 +20,7 @@ public PostgresSQLConnector(boolean debug) {
         try {
             this.establishConnection();
         } catch (SQLException e) {
+            log.error("Constructor", e);
             e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)